R/benchmark-dataframe.R

```diff
@@ -0,0 +1,113 @@
+#' A vector of benchmark attribute names run on `ursa-i9-9960x`
+#'
+#' @export
+URSA_I9_9960X_R_BENCHMARK_NAMES <- c(
```

Comment:
I'm fine with this config looking like this for now (we could turn it into a function once you know more use cases) but maybe it should be in its own file?

Reply:
Yeah, I'm thinking a few things in this file should move, really; all the `run()` stuff should go in `run.R`; `default_params.BenchmarkDataFrame()` should move to be with the rest of the methods (which should probably be split into their own file), and this should go in `config.R` or something, probably. So TODOs:

- [ ] move `run()`
- [ ] move `default_params.BenchmarkDataFrame()`
- [ ] move config

Reply:
done